app/assets/javascripts/initializers/initializeAllFollowButts.js

```diff
@@ -1,10 +1,65 @@
 function initializeAllFollowButts() {
   var followButts = document.getElementsByClassName('follow-action-button');
   for (var i = 0; i < followButts.length; i++) {
-    initializeFollowButt(followButts[i]);
+    if (!followButts[i].className.includes("follow-user")) {
+      initializeFollowButt(followButts[i]);
+    };
   }
 }
 
+function fetchUserButtons(idButtonHash) {
```

Comment:
Consider a different name as this is really fetching whether or not a user is being followed.